Makefile.am

```diff
@@ -913,7 +917,7 @@ test_apps += tests/test_poller \
 	tests/test_scatter_gather \
 	tests/test_dgram \
 	tests/test_app_meta \
-	tests/test_router_notify
+	tests/test_router_notify \
```

Comment:
this will likely break the build and is not necessary anymore

Reply:
Sorry, I pushed to master before checking everything, fixing this now.